core/app/models/spree/order/store_credit.rb

```diff
@@ -10,9 +10,7 @@ def remove_store_credit_payments
       end
 
       def covered_by_store_credit?
-        return false unless user
-
-        user.total_available_store_credit >= total
+        user.present? && !total_minus_store_credits.positive? && total_applied_store_credit.positive?
```

Comment:
@mad-eel can we make this more readable, aka `total_applied_store_credit >= total` ?